jdaviz/configs/imviz/tests/test_parser.py

```diff
@@ -418,29 +421,56 @@ def test_parse_hst_drz(self, imviz_helper):
         assert_allclose(sky.dec.deg, 10.802045612042956, rtol=1e-3)
         data_unit = u.electron / u.s
         assert_quantity_allclose(tbl[0]['background'], 0.0014 * data_unit)
-        assert_quantity_allclose(tbl[0]['sum'], 112.712406 * data_unit, rtol=1e-3)
-        assert_quantity_allclose(tbl[0]['sum_aper_area'], 2583.959958 * PIX2, rtol=1e-3)
-        assert_array_equal(tbl[0]['pixarea_tot'], None)
-        assert_array_equal(tbl[0]['aperture_sum_counts'], None)
-        assert_array_equal(tbl[0]['aperture_sum_counts_err'], None)
-        assert_array_equal(tbl[0]['counts_fac'], None)
-        assert_array_equal(tbl[0]['aperture_sum_mag'], None)
-        assert_array_equal(tbl[0]['flux_scaling'], None)
-        assert_quantity_allclose(tbl[0]['min'], -0.02422 * data_unit, rtol=1e-3)
-        assert_quantity_allclose(tbl[0]['max'], 1.577081 * data_unit, rtol=1e-3)
-        assert_quantity_allclose(tbl[0]['mean'], 0.043684 * data_unit, rtol=1e-3)
-        assert_quantity_allclose(tbl[0]['median'], 0.02129 * data_unit, rtol=1e-3)
-        assert_quantity_allclose(tbl[0]['mode'], -0.023497 * data_unit, rtol=1e-3)
-        assert_quantity_allclose(tbl[0]['std'], 0.09908 * data_unit, rtol=1e-3)
-        assert_quantity_allclose(tbl[0]['mad_std'], 0.023219 * data_unit, rtol=1e-3)
-        assert_quantity_allclose(tbl[0]['var'], 0.009817 * (data_unit * data_unit), rtol=1e-3)
-        assert_quantity_allclose(tbl[0]['biweight_location'], 0.020978 * data_unit, rtol=1e-3)
-        assert_quantity_allclose(tbl[0]['biweight_midvariance'], 0.000606 * (data_unit * data_unit), rtol=1e-3)  # noqa
-        assert_quantity_allclose(tbl[0]['fwhm'], 21.230628 * u.pix, rtol=1e-3)
-        assert_quantity_allclose(tbl[0]['semimajor_sigma'], 9.712415 * u.pix, rtol=1e-3)
-        assert_quantity_allclose(tbl[0]['semiminor_sigma'], 8.260686 * u.pix, rtol=1e-3)
-        assert_quantity_allclose(tbl[0]['orientation'], 80.054019 * u.deg, rtol=1e-3)
-        assert_quantity_allclose(tbl[0]['eccentricity'], 0.525929, rtol=1e-3)
+        # This file keeps changing in reprocessing between two shapes, which gives different values
+        if data.shape[1] == 4219:
+            assert_quantity_allclose(tbl[0]['sum'], 112.712406 * data_unit, rtol=1e-3)
+            assert_quantity_allclose(tbl[0]['sum_aper_area'], 2583.959958 * PIX2, rtol=1e-3)
+            assert_array_equal(tbl[0]['pixarea_tot'], None)
```

Comment:
Can we at least move the same checks out of if-else to reduce code duplication? Thanks!